authui/src/authflowv2/dialog.ts

```diff
@@ -28,8 +30,12 @@ export class DialogController extends Controller {
   };
 
   close = () => {
-    this.element.classList.remove("open");
     this.element.classList.add("close");
+    // We want close animation to show before `display: none` is applied
+    setTimeout(
+      () => this.element.classList.remove("open"),
+      CLOSE_ANIMATION_DURATION_MS
+    );
```

Comment:
It does not seems a proper fix.

Reply:
I tried using `visibility: hidden -> visible` + `transformation` to replace existing `keyframe` + `animation`.

Seems achieved same result; ref [Use visibility over display for dialog animation](https://github.com/authgear/authgear-server/pull/4541/commits/1e69337b328d4ea8cc07ba39c4ce46b6b26f7bfa)

Reply:
okay, resolved